### ISSUE TITLE
Xcode 8.3.1, Cocoapods 1.2.1 doesn't generate the Leanplum framework.

### ISSUE THREAD

**Issue by rafnocx** [Leanplum/Leanplum-iOS-SDK#19]
Hi, 
I'm having an issue integrating Leanplum using Cocoapods.
I have another project that doesn't use framework and it generates just fine.
Running `pod install`, it doesn't add the Leanplum framework as expected.

Below is my `Podfile`:
```
platform :ios, '9.3'
source 'https://github.com/CocoaPods/Specs.git'

# ignore all warnings from all pods
inhibit_all_warnings!

target 'MyApp' do
  use_frameworks!
  pod 'Leanplum-iOS-SDK'
end
```

Am I missing anything? 🤔 
Any help would be greatly appreciated!

**Comment by mkeremkeskin** [comment]
Hi,

I'm having this problem to. Cocoapods shows it is installed but Leanplum is empty.
Does last commit can cause this?

Thnx.

**Comment by bahacan19** [comment]
@mkeremkeskin if this blocks you, you can proceed  "Swift: Manual Setup" way.
https://www.leanplum.com/docs

**Comment by kjacobs-aa** [comment]
Same here

**Comment by gigasmile** [comment]
I pushed it back to how it used to be. Run `pod cache clean --all` before `pod install` and it should work properly.

We're in middle of Open Sourcing and right now every commit has a binary. So if you clone the repository you will download all the binaries, which is quite big. My goal was to remove the binaries from the commit and specify the podspec to fetch the framework directly from the release. This did not work and I rolled it back.
